list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
The code is susceptible to SQL injection, and using raw SQL queries makes it less secure. Shift to prepared statements and refactor the code into an object-oriented design pattern using a Repository for data handling to improve security and maintainability.